edx-platform/pearson-vue-theme/lms/static/sass/partials/lms/theme/_extras.scss

```diff
@@ -323,9 +308,12 @@ iframe.rocket-chat-view {
   display: block;
 }
 
-.login-container {
-   position: absolute;
-   right: 450px;
```

Comment:
@luismorenolopera, Can we keep the same indentation throughout the file?

Reply:
done